doc/whats_new/v0.20.rst

```diff
@@ -246,6 +246,10 @@ Classifiers and regressors
   overridden when using parameter ``copy_X=True`` and ``check_input=False``.
   :issue:`10581` by :user:`Yacine Mazari <ymazari>`.
 
+- Fixed a bug in :class:`linear_model.RidgeCV` where using negative integer 
```

Comment:
What do you mean by this? negative integer -> integer since the bug is mainly about unexpected error when using integer alpha? (negative integer will be rejected right)

Reply:
I had to add a line to raise an error for negative alphas and in the tests I was testing it worked.

Reply:
Your right, both integers raise error.